src/Components/Footer/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+import { useContext } from 'react';
+import classNames from 'classnames';
+
+import { Context } from '../../Context';
+import { Filter } from '../../types/Filter';
+
+export const Footer = () => {
+  const {
+    handleStatusEdit,
+    handleActiveTodos,
+    filter,
+    setFilter,
+  } = useContext(Context);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${handleActiveTodos} items left`}
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames(
+            'filter__link',
+            { selected: filter === Filter.ALL },
+          )}
+          data-cy="FilterLinkAll"
+          onClick={() => setFilter(Filter.ALL)}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames(
+            'filter__link',
+            { selected: filter === Filter.ACTIVE },
+          )}
+          data-cy="FilterLinkActive"
+          onClick={() => setFilter(Filter.ACTIVE)}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={classNames(
+            'filter__link',
+            { selected: filter === Filter.COMPLETED },
+          )}
+          data-cy="FilterLinkCompleted"
+          onClick={() => setFilter(Filter.COMPLETED)}
+        >
+          Completed
+        </a>
+      </nav>
+
+      {/* don't show this button if there are no completed todos */}
```

Comment:
as comment suggest you should show this button only if there are completed todos